modules/apps/frontend-data-set/frontend-data-set-api/src/main/java/com/liferay/frontend/data/set/model/FDSActionDropdownItem.java

```diff
@@ -14,6 +14,17 @@
  */
 public class FDSActionDropdownItem extends DropdownItem {
 
+	public FDSActionDropdownItem(
+		String confirmationMessage, Boolean highlighted, String href,
```

Comment:
`Boolean` will probably not be acceptable to Brian, unless there is a valid case where the value is `null`. If there is no such case, it should be `boolean`.